### ISSUE TITLE
Ticket Logging Shows Incorrect IP

### ISSUE THREAD

**Issue by joshespi** [Provo-City-School-District/help.provo.edu#50]
Ticket logs are now loading into graylog.

we are still having a problem of they displaying with the apache proxy IP for every entry instead of the actual requestor IP.

**found these articles**
https://cylab.be/blog/194/apache-log-real-ip-addresses-behind-a-reverse-proxy

https://confluence.jaytaala.com/display/TKB/Logging+remote+ip+address+when+using+reverse+proxy

tried a minor test but it caused all proxied domains to stop working. i'll try to apply these changes after hours to see if it works. 

Helpdesk Ticket: https://helpdesk.provo.edu/helpdesk/WebObjects/Helpdesk.woa/wa/TicketActions/view?ticket=114570

**Comment by joshespi** [comment]
did some research and found the articles mentioned in the request detail.

did a minor test which caused things to break so i undid those changes.

will do more testing on this after hours.

**Comment by joshespi** [comment]
doing more research i was able to figure out that the changes I needed to make where actually on the destination stack rather than the device being used as a proxy. 

after testing I modified our php dockerfile to make the needed changes to our apache conf when it builds our php image.

I pushed these changes live after additional testing.

code changes
https://github.com/Provo-City-School-District/help.provo.edu/commit/5ebd5fb085cfc6c1c09a7c52ee33282e3834006a

Ticket tracking time:
https://helpdesk.provo.edu/helpdesk/WebObjects/Helpdesk.woa/wa/TicketActions/view?ticket=114570
